core/src/test/scala/org/bykn/bosatsu/EvaluationTest.scala

```diff
@@ -2458,7 +2458,7 @@ test = Assertion(substitute.eq_Int(42), "basis substitution")
   runBosatsuTest(List("""
 package A
 
-three = 2.(x -> add(x, 1))
+three = 2.(x -> add(x, 1))()
```

Comment:
this is kind of weird... seems like it would be better to have:
```scala
def pipe(x, f): f(x)

2.pipe(x -> add(x, 1))
# or
operator |> = pipe
2 |> (x -> add(x, 1))
```